spec/requests/reviewables_controller_spec.rb

```diff
@@ -1021,6 +1021,65 @@ def perform_approve_phony(performed_by, args)
       end
     end
 
+    describe("#scrub") do
```

Comment:
```suggestion
    describe "#scrub" do
```

Tiny nitpick for consistency with all our tests.